cmd/deploy/deploy.go

```diff
@@ -125,6 +127,19 @@ func Deploy(ctx context.Context) *cobra.Command {
 			// deploy command. If not, we could be proxying a proxy and we would be applying the incorrect deployed-by label
 			os.Setenv(model.OktetoSkipConfigCredentialsUpdate, "false")
 			if options.ManifestPath != "" {
+				// we need to store the original manifest path as relative path before switching the cwd
+				options.OriginalManifestPath = options.ManifestPath
+				if filepath.IsAbs(options.ManifestPath) {
+					cwd, err := os.Getwd()
```

Comment:
I would include a comment before this line to be careful of keeping this before doing the change for the working directory. In case of a future refactor that might move this logic